XX.md

```diff
@@ -0,0 +1,73 @@
+NIP-XX: Nostr Verified Podcasts
+======
+
+`draft` `optional`
+
+## Abstract
+
+This NIP defines a method for podcast owners to announce and verify their podcasts on the Nostr network, enabling better discovery and integration of traditional RSS-based podcasts with Nostr-native podcast episodes.
+
+## Introduction
+
+While [NIP-XX Audio Track](https://github.com/nostr-protocol/nips/pull/1043) defines kind 31338 for podcast tracks, many traditional podcasts do not publish these events, making them undiscoverable on the Nostr network. This NIP addresses this issue by introducing podcast announcements and a verification mechanism.
+
+When an npub becomes a verified owner of an RSS feed, kind `31338` podcast events owned by that npub may be directly associated with the verified RSS feed in podcasting clients. In this way, the verification creates a bridge between traditional RSS feeds and nostr-based podcast publishing.
+
+## Podcast Announcements
+
+A podcast owner can publish an announcement on Nostr to make their podcast discoverable:
+
+```json
+{
+  "kind": 1338,
+  "tags": [
+    ["i", "podcast:guid:78ae77a3-b180-5cfa-a7ba-31ac82df8ba9", "https://example.com/podcast/feed.xml"]
```

Comment:
¯\_(ツ)_/¯ idk, would it help? The podcast's GUID never changes, and the feed would change very rarely. They can publish another 1338 if the feed URI did change. No other data is included because the RSS is the authoritative source of metadata for the podcast so having titles and descriptions in the event wouldn't be helpful... so I can't think of a need for a PRE but maybe someone else can...